r-logs.tf

```
@@ -1,13 +1,12 @@
 module "diagnostics" {
   source  = "claranet/diagnostic-settings/azurerm"
-  version = "6.2.0"
+  version = "6.5.0"
```

```suggestion
  version = "~> 6.5.0"
```
We've started to use this notation for all included modules to have fixes propagated quickly.

---

done!